sql/functions-and-operators/aggregate-group-by-functions.md

```diff
@@ -0,0 +1,39 @@
+# Aggregate (GROUP BY) Functions
+
+## Aggregate (GROUP BY) Function Descriptions
+
+see: [https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html](https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html)
```

Comment:
see: [https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html](https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html) ->
This section describes the supported MySQL group (aggregate) functions in TiDB.